packages/web-components/src/mixins/form-field-error.css

```diff
@@ -4,14 +4,16 @@
     margin-bottom: 1.2rem;
 }
 
-:host([error]:not([error=''])) {
+:host([error]:not([error=''])), 
+:host([showerror="true"]) {
     border-left: 0.4rem solid  var(--color-secondary-dark);
     padding-left: 2rem;
     position: relative;
 }
 
 @media screen and (min-width: 1008px) {
-    :host([error]:not([error=''])) {
+    :host([error]:not([error=''])), 
+    :host([showerror="true"]) {
```

Comment:
Now that we're using the `error` prop on the checkbox component, do we still need these changes?

Reply:
Yes, we still need that to display error styling:

![Screen Shot 2022-09-15 at 8 42 28 AM](https://user-images.githubusercontent.com/872479/190419535-c4ec1a83-d60a-4eeb-a75e-2a87668323f0.png)

Reply:
If we force the `error` prop to [reflect into the DOM](https://stenciljs.com/docs/properties#reflect-properties-values-to-attributes-reflect) like this:

```diff
diff --git a/packages/web-components/src/components/va-checkbox/va-checkbox.tsx b/packages/web-components/src/components/va-checkbox/va-checkbox.tsx
index c856c76..ccc8c60 100644
--- a/packages/web-components/src/components/va-checkbox/va-checkbox.tsx
+++ b/packages/web-components/src/components/va-checkbox/va-checkbox.tsx
@@ -32,7 +32,7 @@ export class VaCheckbox {
   /**
    * The error message to render.
    */
-  @Prop() error?: string;
+  @Prop({ reflect: true }) error?: string;
 
   /**
    * The description to render. If this prop exists, va-checkbox will render it
```
Then it seems to look like we would expect without these CSS changes:

![The browser's inspector tab is opened on a local storybook instance for the Privacy Agreement web component. The `error` prop is seen on the checkbox component, and the error styles apply appropriately.](https://user-images.githubusercontent.com/2008881/190466418-0523dcec-47b1-4444-896a-f9beb6d4f7f4.png)

Reply:
Ok so do you want another PR to update the `va-checkbox` prop? Or are you fine with that being in this PR?

Reply:
I think that's fine to include in this PR since it's not a fix for anything independent of this PR.